ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -109,7 +110,7 @@ const NetworksForm = ({
   const [rpcUrl, setRpcUrl] = useState(selectedNetwork?.rpcUrl || '');
   const [chainId, setChainId] = useState(selectedNetwork?.chainId || '');
   const [ticker, setTicker] = useState(selectedNetwork?.ticker || '');
-  const [suggestedTicker, setSuggestedTicker] = useState('');
+  const [suggestedTicker, setSuggestedTicker] = useState([]);
```

Comment:
Should we create a constant called `DEFAULT_SUGGESTED_TICKER` and set it to `[]` so we aren't needing to set it to `[]` in every place it gets reset?

Reply:
kriseng83@gmail.com

Reply:
> Should we create a constant called `DEFAULT_SUGGESTED_TICKER` and set it to `[]` so we aren't needing to set it to `[]` in every place it gets reset?

kriseng83@gmail.com

Reply:
> Should we create a constant called `DEFAULT_SUGGESTED_TICKER` and set it to `[]` so we aren't needing to set it to `[]` in every place it gets reset?

Yes

Reply:
> Should we create a constant called `DEFAULT_SUGGESTED_TICKER` and set it to `[]` so we aren't needing to set it to `[]` in every place it gets reset?

Yes

Reply:
> Should we create a constant called `DEFAULT_SUGGESTED_TICKER` and set it to `[]` so we aren't needing to set it to `[]` in every place it gets reset?